spec/prompt_spec.rb

```diff
@@ -6,44 +6,85 @@
 
 describe Prompt do
   let(:board) { Board.new }
+  subject(:prompt) { described_class.new(board) }
 
-  describe 'Correct Terminal Output' do
-    it 'Should Print the welcome message' do
-      expect { Prompt.welcome }.to output(Message.welcome).to_stdout
+  describe '#welcome' do
+    it 'should Print the welcome message' do
+      expect { prompt.welcome }.to output(Message.welcome).to_stdout
     end
+  end
 
-    it 'Should Print the tic tac toe board' do
-      expect { Prompt.print_board }.to output(board.display_board).to_stdout
+  describe '#print_board' do
+    it 'should Print the tic tac toe board' do
+      expect { prompt.print_board }.to output(board.display_board).to_stdout
```

Comment:
Looking at this test, I wonder if maybe the board knows how to do too much? i.e. it knows how to represent itself for some interface as well as all the board logic.